packages/headless/src/api/search/facet-search/category-facet-search/category-facet-search-request.test.ts

```diff
@@ -41,7 +41,7 @@ describe('#buildCategoryFacetSearchRequest', () => {
     const query = 'hello';
     state.categoryFacetSearchSet[id].options.query = query;
 
-    expect(buildParms().query).toEqual(query);
+    expect(buildParms().query).toEqual('*' + query + '*');
```

Comment:
Suggestion: 
```suggestion
    expect(buildParms().query).toEqual(`*${query}*`);
```

Also seems like there's a typo in that test file that you could fix, it should be `buildParams`